SwiftWisdom/Core/StandardLibrary/Array/Array+Utilities.swift

```diff
@@ -54,3 +54,43 @@ extension Array {
         }
     }
 }
+
+extension CollectionType {
+    public var ip_middleValues: [Generator.Element]? {
```

Comment:
Let's field some external opinions on this one as well to see if we can get some opinions one way or the other.